crates/cranelift/src/obj.rs

```diff
@@ -198,12 +199,30 @@ impl<'a> ModuleTextBuilder<'a> {
                             object::write::Relocation {
                                 symbol,
                                 flags,
-                                offset: off + u64::from(r.offset),
+                                offset: reloc_offset,
                                 addend: r.addend,
                             },
                         )
                         .unwrap();
                 }
+
+                // This relocation is used to fill in which hostcall signature
+                // is desired within the `call_indirect_host` opcode of Pulley
+                // itself. The relocation target is the start of the instruction
+                // and the goal is to insert the static signature number, `n`,
+                // into the instruction.
+                //
+                // At this time the instructions is 4-bytes large (3 bytes for
+                // the opcode, one for the one-byte payload), so we target the
+                // third byte. The value `n` here should always fit within a
```

Comment:
```suggestion
                // fourth byte. The value `n` here should always fit within a
```

Or describe which 0-based index.